pkg/workload/workload.go

```diff
@@ -156,12 +203,18 @@ func ResourceQuantity(name corev1.ResourceName, v int64) resource.Quantity {
 	}
 }
 
-func (r Requests) scale(f int64) {
+func (r Requests) scaleUp(f int64) {
 	for name := range r {
 		r[name] *= f
 	}
 }
 
+func (r Requests) scaleDown(f int64) {
+	for name := range r {
+		r[name] /= f
+	}
+}
+
 // UpdateStatus updates the condition of a workload with ssa,
 // filelManager being set to managerPrefix + "-" + conditionType
 func UpdateStatus(ctx context.Context,
```

Comment:
Can you rename to ApplyStatus?

Reply:
Let's keep "Apply" , for when we only push parts of a workload to the api server.